feat: Update distributed authority overview (#1454)
Unity-Technologies/com.unity.multiplayer.docs: docs/terms-concepts/distributed-authority.md

:::

## Considerations

Using a distributed authority topology is typically not suitable for high-performance competitive games that require an accurate predictive motion model. The distributed authority model successfully addresses a lot of visual and input-related issues, but does have some limitations:

* Since authority and ownership of objects is distributed across clients, there's typically no single physics simulation governing the interaction of all objects. This can require approaching physics-related gameplay differently compared to a traditional client-server context.
* Depending on the platform and overall design of your product, it can be easier for bad actors to cheat. The authority model gives more trust to individual clients. Evaluate your cheating tolerance when developing with distributed authority.

## Session ownership

> When using the distributed authority topology, it is necessary to have a single dedicated client who is responsible for managing and synchronizing global game state-related tasks. This client is referred to as the session owner.

Reviewer: ```suggestion
When using the distributed authority topology, it's necessary to have a single dedicated client that's responsible for managing and synchronizing global game state-related tasks. This client is referred to as the session owner.
```

> The initial session owner is the first client who joins when the session is created. If this client disconnects during the game, a new session owner will be automatically selected and promoted from within all the clients who are currently connected.

Reviewer: ```suggestion
The initial session owner is the first client that joins when the session is created. If this client disconnects during the game, a new session owner is automatically selected and promoted from within the clients that are currently connected.
```
